src/Components/ADT3DSceneBuilder/Internal/BuilderLeftPanel/BuilderLeftPanel.tsx

```diff
@@ -179,94 +182,113 @@ const BuilderLeftPanel: React.FC<IBuilderLeftPanelProps> = ({ styles }) => {
         [dispatch]
     );
 
-    const onCreateElementClick = () => {
-        dispatch({
-            type: SET_ADT_SCENE_BUILDER_SELECTED_ELEMENT,
-            payload: null
-        });
+    // START of scene element related callbacks
+    const setSelectedElement = useCallback(
+        (element: ITwinToObjectMapping) => {
+            dispatch({
+                type: SET_ADT_SCENE_BUILDER_SELECTED_ELEMENT,
+                payload: element
+            });
+        },
+        [dispatch]
+    );
+
+    const onCreateElementClick = useCallback(() => {
+        setSelectedElement(getDefaultElement({ id: v4() }));
```

Comment:
I kind of liked using a utils function for this instead of v4 directly, kinda like your other PR that adds v4 to the utils.  do that plz prolly?

Reply:
sure. done.